src/main/java/com/sda/arppl2/Piesek.java

```diff
@@ -0,0 +1,45 @@
+package com.sda.arppl2;
+
+public class Piesek {
+
+    private String imie;
+    private int wiek;
+    private String kolor;
+
+    public String getImie() {
```

Comment:
W tym zadaniu wydaje mi się że wystarczyło zdefiniować tylko 1 obiekt zawierający kilka właściwości. Ciekawie że zrobiłeś to w inny sposób. Jeśli i tak działa, akceptuje :)